src/iopae.cpp

```diff
@@ -1317,17 +1319,15 @@ void PaeInput::parseNote(pae::Note *note)
             m_measure->AddChild(trill);
         }
 
-        if (m_last_tied_note != NULL) {
-            mnote->SetTie(TIE_t);
-            m_last_tied_note = NULL;
+        if (m_tie != NULL) {
+            m_tie->SetEndid(mnote->GetUuid());
+            m_measure->AddChild(m_tie);
+            m_tie = NULL;
         }
 
         if (note->tie) {
-            if (mnote->GetTie() == TIE_t)
-                mnote->SetTie(TIE_m);
-            else
-                mnote->SetTie(TIE_i);
-            m_last_tied_note = mnote;
+            m_tie = new Tie();
```

Comment:
You need to check in the PaeInput destructor if `m_tie == NULL` and delete it if not, otherwise there will be a memory leak.